content/news/2024-05-14-everse-workshop/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "EVERSE Satellite meeting (International Research Software Funders Workshop)"
+draft: false
+date: 2024-05-14T00:00:00Z
+start_date: 2024-05-14T00:00:00Z
+end_date: 2024-05-14T00:00:00Z
+summary: >
+    The main goal of the workshop is to serve as a platform for EVERSE to showcase the work of the project, and receive input from all relevant stakeholders, while facilitating networking and collaboration opportunities between all participants. Stakeholders of particular relevance in the workshop are Funders and Policy Makers, in order to increase awareness on the challenges and opportunities around research, as well as relevant community projects. The expectation is for participants to bring with them their communities’ best practices and services on research software (including but limited tos quality, management, sustainability, excellence, etc) in order to engage with EVERSE, while also establishing a bidirectional channel of communication moving forward.
```

Comment:
"including but limited tos quality" is weird - should it be "including, but not limited to, quality..."

"etc" should be "etc."